concepticondata/references/references.bib

```diff
@@ -3150,4 +3150,17 @@ @article{Ruhlen1998
   pages        = {13994-13996},
   year         = {1998},
   publisher    = {National Academy of Sciences}
-}
\ No newline at end of file
+}
+
+@article{Lee2015,
+  doi          = {10.1371/journal.pone.0128448},
+  url          = {https://doi.org/10.1371/journal.pone.0128448},
```

Comment:
Nitpicking here, but if the URL is the DOI, I'd pass it as DOI only.

Reply:
ok, so remove 'url'?

Reply:
done